client/src/App.scss

```diff
@@ -51,6 +51,14 @@ h1 {
   padding-bottom: 0 !important;
 }
 
+.pl-0 {
+  padding-left: 0 !important;
```

Comment:
be careful when using `!important`! are you overriding default classes? 

what does `pl` and `mt` stand for? 

i suggest instead of rewriting classes, assign a new class name, something meaningful such as ".card_primary_line"

Reply:
The codes with p = padding and m = margin. I have these codes to either remove the p/m for alignment or to add a bit more (0 for removing and 1/2 for adding more). This is used for aligning specific elements that have either too much or either one. Should I do this another way?

Reply:
oh! I see . 

The convention you are approaching is very much like creating a style guide. 

When creating these general class names, I would keep in mind
1. Do I have a general style guide I'd like to follow? 
2. Document what your naming convnetions are like `p = padding m = margin` etc. It's not very clear and therefore could create a lot of confusion
3. Is the `!important` required? 

I think this is okay if these are styles you use often. 
Otherwise, I would avoid using !important

Reply:
Will do, thank you!